Analyzer/src/Semra_Analyzer.cc

```diff
@@ -121,35 +139,154 @@ void Semra_Analyzer::Loop(NTupleReader& tr, double weight, int maxevents, bool i
             }
             
             //PileupWeight = tr.getVar<double>("_PUweightFactor");
-            bTagScaleFactor   = tr.getVar<double>("bTagSF_EventWeightSimple_Central");
+            bTagScaleFactor      = tr.getVar<double>("bTagSF_EventWeightSimple_Central");
             htDerivedScaleFactor = tr.getVar<double>("htDerivedweight");
-            topPtScaleFactor = tr.getVar<double>("topPtScaleFactor");
+            topPtScaleFactor     = tr.getVar<double>("topPtScaleFactor");
             prefiringScaleFactor = tr.getVar<double>("prefiringScaleFactor");
-            puScaleFactor = tr.getVar<double>("puWeightCorr");
+            puScaleFactor        = tr.getVar<double>("puWeightCorr");
             
             weight *= eventweight*leptonScaleFactor*bTagScaleFactor*htDerivedScaleFactor*prefiringScaleFactor*puScaleFactor;
         }
-        
 
-        // SEMRA / Make cuts and fill histograms here & cutmap
-        // cutmap
+	
+	// -------------------------------------------------
+	// -- Calculate non-top (2 neutralinos) mass & pT
+	// -------------------------------------------------
+        double nonTopMass = 0;
+	const double Xcut_low = 100;
+        const double Xcut_high = 200;
+	for(int ijet = 0; ijet < Jets.size(); ijet++) {
+		if (!GoodJets_pt45[ijet] && !GoodBJets_pt45[ijet]) {
```

Comment:
I don't understand this line of code. I don't think it does what you think it does.